api/src/main/java/org/pmiops/workbench/db/dao/InstitutionEmailAddressDao.java

```diff
@@ -0,0 +1,16 @@
+package org.pmiops.workbench.db.dao;
+
+import java.util.List;
+import org.pmiops.workbench.db.model.DbInstitution;
+import org.pmiops.workbench.db.model.DbInstitutionEmailAddress;
+import org.springframework.data.repository.CrudRepository;
+
+public interface InstitutionEmailAddressDao
+    extends CrudRepository<DbInstitutionEmailAddress, Long> {
+
+  List<DbInstitutionEmailAddress> findAllByInstitution(DbInstitution institution);
+
+  default void deleteAllByInstitution(DbInstitution institution) {
```

Comment:
So does this operation leave users with those email addresses in a bad state? Do/should they still have institutions or not?

Reply:
Existing users will never be affected by any changes to these tables.  The Launch MVP version of Verification is strictly a one-time-only event at registration.  The Institution could go away entirely and the user would still be enabled.

Obviously there's a whole world of verification *maintenance* that is not addressed here.  We are nowhere close to having an agreed-upon set of requirements for those stories.  It may not even be the RW that does this.